src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -212,121 +212,121 @@
-    <sys:String x:Key="Activate_Uninstall_All_Licenses.Description">Disinstalla tutte le licenze installate. Sarà necessario installare una licenza dopo questa operazione.</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Licenses.Description">Disinstalla tutte le licenze installate. Dopo questa operazione sarà necessario installare una licenza.</sys:String>
     <sys:String x:Key="Activate_Uninstall_All_Licenses_Button">Disinstalla</sys:String>
 
-    <sys:String x:Key="Activate_Keys_Management">Gestione delle chiavi</sys:String>
+    <sys:String x:Key="Activate_Keys_Management">Gestione chiavi</sys:String>
     <sys:String x:Key="Activate_Keys_Management.Description">Installa, disinstalla le chiavi o attiva Office offline.</sys:String>
-    <sys:String x:Key="Activate_Product_Key">Product key</sys:String>
-    <sys:String x:Key="Activate_Product_Key.Description">Installa product key di Office, formato della product key: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
+    <sys:String x:Key="Activate_Product_Key">Chiave prodotto</sys:String>
+    <sys:String x:Key="Activate_Product_Key.Description">Installa chiave prodotto di Office, formato chiave prodotto: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
     <sys:String x:Key="Activate_Install_Product_Key">_Installa chiave</sys:String>
     <sys:String x:Key="Activate_Uninstall_Unactivated_Key">Disinstalla t_utte le chiavi non attivate</sys:String>
     <sys:String x:Key="Activate_Uninstall_All_Product_Keys">Disinst_alla tutte le chiavi</sys:String>
-    <sys:String x:Key="Activate_Install_CID">Installa confirmation ID</sys:String>
-    <sys:String x:Key="Activate_Install_CID.Description">Installa confirmation ID per attivare Office offline.</sys:String>
-    <sys:String x:Key="Activate_Input_CID">Inserire il confirmation ID</sys:String>
+    <sys:String x:Key="Activate_Install_CID">Installa ID conferma</sys:String>
+    <sys:String x:Key="Activate_Install_CID.Description">Installa ID conferma per attivare Office offline.</sys:String>
+    <sys:String x:Key="Activate_Input_CID">Inserisci ID conferma</sys:String>
 
     <sys:String x:Key="Activate_KMS_Management">Gestione KMS</sys:String>
     <sys:String x:Key="Activate_KMS_Management.Description">Imposta o reimposta l'host o la porta del KMS.</sys:String>
     <sys:String x:Key="Activate_KMS_Apply">Impo_sta host</sys:String>
     <sys:String x:Key="Activate_KMS_Host">Host KMS</sys:String>
-    <sys:String x:Key="Activate_KMS_Host.Description">Nome host KMS. Per esempio, kms.example.com, kms.example.com:1688 o indirizzo IP.</sys:String>
-    <sys:String x:Key="Activate_KMS_Host_Cache.Enable">Abilita la memorizzazione nella cache dell'host</sys:String>
-    <sys:String x:Key="Activate_KMS_Host_Cache.Disable">Disabilita la memorizzazione nella cache dell'host</sys:String>
+    <sys:String x:Key="Activate_KMS_Host.Description">Nome host KMS. Per esempio, kms.esempio.com, kms.esempio.com:1688 o indirizzo IP.</sys:String>
```

Comment:
After thinking about it, I still think that domain names should not be translated. Keeping `kms.example.com` is better.

Reply:
@YerongAI 

Updated.
Please check if it's ok for you.